src/Dialogs/dlgChecklist.cpp

```
@@ -133,8 +134,17 @@ dlgChecklistShowModal()
   if (checklist.empty())
     checklist.emplace_back(ChecklistPage{
         _("No checklist loaded"),
-        _("Create a checklist file (e.g. checklist.xcc) or select one in Site "
-          "Files > Checklist."),
+        _("# Getting Started\n\n"
+          "Download the example checklist or create your own:\n\n"
+          "- [Download Example](https://xcsoar.org/download/data/xcsoar-checklist.txt)\n"
+          "- [View Documentation](https://xcsoar.org/develop/checklist.html)\n\n"
+          "Then select it in **Site Files > Checklist**.\n\n"
```

Cppcheck (2.19.0)

[error] Dereference of an invalid iterator

(derefInvalidIterator)